src/test/ui/closures/2229_closure_analysis/migrations/macro.stderr

```diff
@@ -0,0 +1,24 @@
+error: changes to closure capture in Rust 2021 will affect drop order
+  --> $DIR/macro.rs:10:13
+   |
+LL |     let _ = || dbg!(a.0);
+   |             ^^^^^^^^---^
+   |                     |
+   |                     in Rust 2018, closure captures all of `a`, but in Rust 2021, it only captures `a.0`
+...
+LL | }
+   | - in Rust 2018, `a` would be dropped here, but in Rust 2021, only `a.0` would be dropped here alongside the closure
```

Comment:
Pre-existing: I think saying 'as part as the closure' would make more sense than 'alongside the closure'.

Reply:
https://github.com/rust-lang/rust/pull/87984